plugins/org.eclipse.papyrus.uml.interaction.graph/src/org/eclipse/papyrus/uml/interaction/graph/Vertex.java

```diff
@@ -100,7 +99,6 @@ default boolean precedes(Vertex other) {
 	 * @return whether I succeed the {@code other} in the interaction
 	 */
 	default boolean succeeds(Vertex other) {
-		predecessors().collect(Collectors.toList());
```

Comment:
Oops!  That was for debugging with ⌘I.

Reply:
Yeah, I only found it, because I added exactly the same in the method above while debugging. :) One slight disadvantage of streams.